cdci_data_analysis/analysis/instrument.py

```diff
@@ -69,26 +67,20 @@ def __init__(self,
                  data_server_query_class=None,
                  query_dictionary={}):
 
-        #name
-        self.name=instr_name
-
+        # name
+        self.name = instr_name
+        # logger
+        self.logger = logging.getLogger(repr(self))
```

Comment:
we should be careful that this does not lead to unconstrained growth of loggers, for every request. rigth now, it happens in other case - see logging_tree output.

just general point, here it might be fine

Reply:
Reason why I did this is because there were spme _print_ within the class. So it is something to keep an eye in the future for sure, but currently I used it only to replace the print that were already there.

Reply:
of course, and it makes sense. 
but note an example of this: https://github.com/oda-hub/dispatcher-app/blob/master/cdci_data_analysis/flask_app/dispatcher_query.py#L269
 it can be avoided though

Reply:
That looks like a proper _logger_ configuration, so perhaps some reformatting of the code and in particular on how the logger is initiated (at least for the _dispatcher_quey_) is needed.

Reply:
#33  opened for the purpose